include/temoto_robot_manager/robot_config.h

```diff
@@ -155,6 +161,7 @@ class RobotConfig
   FeatureNavigation feature_navigation_;
   FeatureGripper feature_gripper_;
   std::map<std::string, FeatureCustom> m_feature_custom_;
+  std::map<std::string, FeatureCommon> m_feature_common_;
```

Comment:
I think the common procedures should not be mixed with features, because it's not a feature, it's a ... `common procedure`